fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -2078,6 +2089,9 @@ private CompletableFuture<Void> run() {
             }
 
             final KeyExpression recordCountKey = getRecordMetaData().getRecordCountKey();
+            // Note: The RecordCountKey may be disabled. But if it is, then the data should be clear, and there is no
```

Comment:
This feels less defensible than in the other method. I think you're right that the clear is ultimately harmless, but it seems out of line with how we treat disabled indexes